src/reporters/errors/templates.js

```diff
@@ -14,6 +15,11 @@ function getMsgPrefix (err, category) {
     `;
 }
 
+function indentMultilineMsg (str, whitespaces) {
+    //NOTE: we remove indentation before first line, because there is already in template.
+    return indentString(str, ' ', whitespaces).replace(new RegExp(`^\\s{${whitespaces}}`), '');
```

Comment:
Whitespace count is always equals to 12. Let's move it to the constant, along with the RegExp